lib/dal/src/management/mod.rs

```diff
@@ -321,20 +653,38 @@ impl<'a> ManagementOperator<'a> {
     }
 
     async fn updates(&mut self) -> ManagementResult<()> {
-        if let Some(updates) = &self.operations.update {
-            for (placeholder, operation) in updates {
-                let component_id = self.get_real_component_id(placeholder).await?;
+        let updates = self.operations.update.take();
+        let Some(updates) = &updates else {
+            return Ok(());
+        };
 
-                if let Some(properties) = &operation.properties {
-                    update_component(self.ctx, component_id, properties, &[]).await?;
+        for (placeholder, operation) in updates {
+            let component_id = self.get_real_component_id(placeholder).await?;
+
+            if let Some(properties) = &operation.properties {
+                update_component(self.ctx, component_id, properties, &[]).await?;
+            }
+
+            if let Some(update_conns) = &operation.connect {
+                if let Some(add_conns) = &update_conns.add {
+                    for to_add in add_conns {
+                        self.create_connection(component_id, to_add).await?;
+                    }
                 }
-                if let Some(raw_geometry) = &operation.geometry {
-                    let mut component = Component::get_by_id(self.ctx, component_id).await?;
-                    component
-                        .set_raw_geometry(self.ctx, raw_geometry.to_owned())
-                        .await?;
+
+                if let Some(remove_conns) = &update_conns.remove {
+                    for to_remove in remove_conns {
+                        self.remove_connection(component_id, to_remove).await?;
+                    }
                 }
```

Comment:
Does it matter if you add before remove or vice versa?

Reply:
Hmm that's a good question. I guess if you wanted to "reconnect" we'd want to ensure removes happened before adds